src/mongo/schemas/FeedbackChannels.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model as createModel } from "mongoose";
+
+export interface IFeedbackChannels {
```

Comment:
Rename the interface, file, schema & model to `FeedbackChannel` instead of `FeedbackChannels`